src/analysis/data_analyzer.rs

```diff
@@ -209,145 +258,145 @@ pub fn analyze_datax86(
                 };
                 queue.push_front(new_analysis);
                 return AnalysisResult::End;
-			}
-			// 2) if not, db <BYTES>, fill bytes 
-			else {
-			    let mut instructions: Instruction<X86Detail>;
-				instructions = Instruction::new();
-				instructions.address = offset as u64;
-				instructions.offset = offset as u64;
-				let bytes = transmute_integer_to_vec(value as usize, address_size);
-				for (i, value) in bytes.iter().enumerate()
-				{
-					instructions.bytes[i] = *value;
-				}
-				instructions.mnemonic = String::from("db");
-				match address_size
-				{
-					2=>instructions.op_str = format!("0x{:x}", value as u16),
-					4=>instructions.op_str = format!("0x{:x}", value as u32),
-					8=>instructions.op_str = format!("0x{:x}", value as u64),
-					_=>{},
-				}
-				instructions.size = address_size;
-				let mut _valid_instr = InstructionInfo
-				  {
-				    instr: instructions,
-				    detail: Vec::new(),
-				};
-				analysis.instr_info.insert(offset as u64, _valid_instr);
-			}
-		},
-	}
-	return AnalysisResult::Continue;
+            }
+            // 2) if not, db <BYTES>, fill bytes 
+            else {
+                let mut instructions: Instruction<X86Detail>;
+                instructions = Instruction::new();
+                instructions.address = offset as u64;
+                instructions.offset = offset as u64;
+                let bytes = transmute_integer_to_vec(value as usize, address_size);
+                for (i, value) in bytes.iter().enumerate()
+                {
+                    instructions.bytes[i] = *value;
+                }
+                instructions.mnemonic = String::from("dd");
```

Comment:
did this change from db to db on purpose?

Reply:
Yeah, it should be DD = Dword, DB = Byte